index.js

```diff
@@ -27,6 +27,10 @@ app.get('/random_ten', (req, res) => {
   res.json(randomTen());
 });
 
+app.get('/jokeByType/:type', (req, res) => {
```

Comment:
Usually, route paths aren't camelCased. I'm thinking `/jokes/:type/random`, ``/jokes/:type/ten`, etc would be more intuitive.